styles.css

```diff
@@ -281,6 +281,11 @@ iframe {
   justify-content: flex-start;
 }
 
+.badge-count .bg-away {
+  position: relative;
+  z-index: 99999999999;
```

Comment:
this much z-index value is not good practice to give. You should not give this much high value